source/Applications/Basic/Visualization/CaptureHDRVisNormals/CaptureHDRVisNormals.cpp

```diff
@@ -0,0 +1,143 @@
+/*
+This example shows how to capture Zivid point clouds, with color and normals.
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <pcl/point_types.h>
+#include <pcl/visualization/cloud_viewer.h>
+
+#include <iostream>
+#include <thread>
+
+using namespace std::chrono_literals;
+
+namespace
+{
+    void visualize(pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr cloud,
```

Comment:
Not sure, but maybe we should ads PCL or Pcl in the name of this function.
Also, does it visualize only normals or normals and point cloud? If both, maybe consider adding that to the name?

visualizePointCloudWithNormalsPCL for example.

Reply:
It visualizes both point cloud and normals, so maybe visualizePointCloudAndNormalsPCL